### DISCUSSION TITLE
Extreme EX-OS support

### DISCUSSION THREAD

**Discussion by gerivives** [hellt/vrnetlab#265] [Ideas]
Dear Community,

They say you loose nothing for asking, so... I believe it would be a great idea to have Extreme EX-OS support in Containerlab natively. I would go on to do it myself but I am not an expert in this topic, thus I am just suggesting it.

If anyone else may be interested, let others know by posting here :).

Thanks!

**Comment by rafabr** [comment]
I've opened a PR to add support for EXOS (#271). Give it a try and let me know if it works for you!
